next/components/thumbprint-components/props-table/prop-type.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
```

Comment:
is this file copied from the old gatsby docs? do we still need to support proptypes if all components are written in TS now?

Reply:
Yeah this one is copied over.